xbmc/video/VideoInfoTag.h

```diff
@@ -47,6 +47,16 @@ struct SActorInfo
   int        order;
 };
 
+class CRating
+{
+public:
+  CRating() { CRating(0.0, 0); }
```

Comment:
I'm pretty sure that this doesn't do what you expect. Please use constructor initialization like
```
CRating()
  : CRating(0.0f, 0)
{ }
```